pkg/capacity/store.go

```diff
@@ -53,15 +53,17 @@ func NewHTTPStore(url string) *HTTPStore {
 }
 
 // Register sends the capacity information to BCDB
-func (s *HTTPStore) Register(nodeID pkg.Identifier, c Capacity, d dmi.DMI, disks Disks) error {
+func (s *HTTPStore) Register(nodeID pkg.Identifier, c Capacity, d dmi.DMI, disks Disks, isVM bool) error {
 	x := struct {
 		Capacity Capacity `json:"capacity"`
 		DMI      dmi.DMI  `json:"dmi"`
 		Disks    Disks    `json:"disks"`
+		IsVM     bool     `json:"is_vm"`
 	}{
 		Capacity: c,
 		DMI:      d,
 		Disks:    disks,
+		IsVM:     isVM,
```

Comment:
Since you modify the structure we sent to BCDB, you also need to add this field into the struct that will be used to deserialize the request server side, which happens here:
https://github.com/threefoldtech/zos/blob/acf49fbc5780985fcba995838606fe8a35ab852f/tools/bcdb_mock/node_handlers.go#L127-L131
And ultimately add it into this type:
https://github.com/threefoldtech/zos/blob/acf49fbc5780985fcba995838606fe8a35ab852f/pkg/gedis/types/directory/tfgrid_node_2.go#L58-L64